### ISSUE TITLE
The API version 2.5.207 does not match the Worker version 2.7.570

### ISSUE THREAD

**Issue by oddvernes** [arkokoley/pdfvuer#93]
It looks like there is a version mismatch between the pdfjs compiled in pdfvuer.umd.js and the pdfjs that gets installed as a dependency. 
pdf.worker.js (which is loaded from the dependency) checks the version number in the message (from pdfvuer.umd) and throws an error.

**Comment by micimize** [comment]
Pinning the pdfjs dist works as a stopgap `"pdfjs-dist": "2.5.207",`

**Comment by micimize** [comment]
@arkokoley I guess that would work but I think ideally it would just be vendored

**Comment by Uraharadono** [comment]
Just installed  `"pdfvuer": "^1.8.3",` and while trying advanced example I get error: 

```
UnknownErrorException {
message: "The API version \"2.5.207\" does not match the Worker version \"2.8.335\".", 
name: "UnknownErrorException", 
details: "Error: The API version \"2.5.207\" does not match the Worker version \"2.8.335\"."
}
details: "Error: The API version \"2.5.207\" does not match the Worker version \"2.8.335\"."
message: "The API version \"2.5.207\" does not match the Worker version \"2.8.335\"."
name: "UnknownErrorException"
__proto__: Error
```

So I guess this corresponds to what you guys said. What version is not affected by this, so I can downgrade to it?

**Comment by micimize** [comment]
@Uraharadono based on your error message looks like it's still `2.5.207`

**Comment by micimize** [comment]
@Uraharadono I don't think you need to downgrade, try just pinning the dist like I mentioned above

> Pinning the pdfjs dist works as a stopgap `"pdfjs-dist": "2.5.207",`

**Comment by Uraharadono** [comment]
@micimize Hey Michael, 
Thanks a lot for your help. I was tired today and I missed totally that I need to install `pdfjs-dist `as well. 
So, after installing both 

    "pdfjs-dist": "2.5.207",
    "pdfvuer": "^1.8.3",
    
It works now. Even with the URL that points to download Action on server. 
Thanks a million times mate.

**Comment by arkokoley** [comment]
Thanks to @micimize @oddvernes for bringing this issue to notice. Fixed this as part of [v1.9.1](https://www.npmjs.com/package/pdfvuer/v/1.9.1)
